crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__RUF027_RUF027.py.snap

```diff
@@ -0,0 +1,183 @@
+---
+source: crates/ruff_linter/src/rules/ruff/mod.rs
```

Comment:
I think it would be great to have some more examples with
* triple quoted strings (multiline and single line)
* single quoted, multiline strings
* Implicitly concatenated strings (if you want to have fun, with comments in between)
* string literals that use escaped characters (and `{` for something other than placeholders)